src/components/scenes/Staking/StakeOverviewScene.tsx

```diff
@@ -149,7 +147,14 @@ const StakeOverviewSceneComponent = (props: Props) => {
         />
       )}
       <MainButton
-        label={unstakeText}
+        label={lstrings.stake_unstake_claim}
+        disabled={!stakePosition?.canUnstakeAndClaim}
```

Comment:
This would violate the strict-boolean rule if we ever enable it. Better do `const {canUnstakeAndClaim = true} = stakePosition`, and then the `canUnstakeAndClaim` variable will work with `!` safely.

Or maybe the default is `false`. It's hard to reason about the way you have it.